resources/external_features_configs/K562_external_features.tsv

```diff
@@ -0,0 +1,28 @@
+input_col	source_col	aggregate_function	join_by	source_file
+averageCorrWeighted	averageCorrWeighted	mean	TargetGene	resources/external_features/EnhancerWeightedCorr_acrossCellTypes_RefSeqCurated.tsv
+H3K4me3_e_max_L_8	H3K4me3_e_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K4me3_e_grad_max_L_8	H3K4me3_e_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K27ac_e_grad_max_L_8	H3K27ac_e_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+DNase_e_grad_max_L_8	DNase_e_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K4me3_e_grad_min_L_8	H3K4me3_e_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K27ac_e_grad_min_L_8	H3K27ac_e_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+DNase_e_grad_min_L_8	DNase_e_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K4me3_p_max_L_8	H3K4me3_p_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K4me3_p_grad_max_L_8	H3K4me3_p_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K27ac_p_grad_max_L_8	H3K27ac_p_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+DNase_p_grad_max_L_8	DNase_p_grad_max_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K4me3_p_grad_min_L_8	H3K4me3_p_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+H3K27ac_p_grad_min_L_8	H3K27ac_p_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+DNase_p_grad_min_L_8	DNase_p_grad_min_L_8	max	overlap	resources/external_features/K562/GraphRegLR_EG_preds_K562_hg38_FDR_1_L_8_saliency_with_features.tsv.gz
+EpiMapScore	Score	mean	overlap	/oak/stanford/groups/engreitz/Projects/Benchmarking/Predictors/EpiMap/processed/linking_collated_BSS00762.bed.gz
+glsCoefficient	GLScoefficient	mean	overlap	/oak/stanford/groups/engreitz/Users/agschwin/distal_regulation_paper/predictors/eg_correlation_predictors/results/ABC_samples/correlation.DHS_tagAlign.100bp.tsv.gz

```

Comment:
Can we avoid oak filepaths here?

Reply:
see above-- even in this case, the relative paths (resources/external_features/K562/GraphReg...) isn't uploaded here as it's too large. we could potentially make a "chr22" version of all the extended features to use as an example and have the full files hosted elsewhere. don't have the bandwidth to do that at the moment though, so what's the best thing to do in the mean time (so the formatting of how to handle external features is available)?